src/Pages/Main/Main.js

```diff
@@ -2,8 +2,51 @@ import React from "react";
 import "./Main.scss";
 
 class Main extends React.Component {
+  constructor() {
+    super();
+
+    this.state = {
+      mainData: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch("http://localhost:3000/data/commonData.json")
```

Comment:
```suggestion
    fetch("/data/commonData.json")
```
3000번 포트를 지정하지 않더라도 이렇게 작성하더라도 같은 의미를 갖습니다.